papermario/globals.inc

```diff
@@ -1631,1678 +1631,590 @@
 .global EnterSingleDoor
 .global ExitDoubleDoor
 .global EnterDoubleDoor
-
```

Comment:
Is this an actual diff or just something weird because of where the branch started from?